homeassistant/components/climate/velbus.py

```diff
@@ -0,0 +1,81 @@
+"""
+Support for Velbus thermostat.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/climate.velbus/
+"""
+import logging
+
+from homeassistant.components.climate import (
+    SUPPORT_OPERATION_MODE, SUPPORT_TARGET_TEMPERATURE, ClimateDevice)
+from homeassistant.components.velbus import (
+    DOMAIN as VELBUS_DOMAIN, VelbusEntity)
+from homeassistant.const import ATTR_TEMPERATURE
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['velbus']
+
+OPERATION_LIST = ['comfort', 'day', 'night', 'safe']
```

Comment:
This is wrong. Only states imported from the base climate component is allowed.

Reply:
I don't know the integration or what those modes do, so I don't have a suggestion at the moment.

Reply:
basically those 4 are are all temperature presets

Reply:
The home assistant operation state explains how the thermostat tries to adjust temperature, eg heat, cool, auto, eco. It doesn't say what temperature the thermostat should set. We have `target_temperature` and `set_temperature` for that.

If I understand correctly `'comfort', 'day', 'night', 'safe'` are levels of temperature the thermostat should try to keep? If so, we shouldn't use operation mode in home assistant to change those.

Reply:
so this means i will only have on operation mode in HA (heat)
and how do i handle the temperature levels?

Reply:
If we can set a specific temperature degree with `set_temperature` these modes are not required, right? I don't think we currently have any matching control options for these modes.